dev/archery/archery/cli.py

```diff
@@ -807,13 +807,15 @@ def release(obj, src, jira_cache):
 
 @release.command('curate')
 @click.argument('version')
+@click.option('--minimal/--full', '-m/-f',
```

Comment:
This looks weird: should I pass `--minimal` or `--full` to get the desired effect? Especially as the default is... False (what does it mean?).

Reply:
I agree it looks a bit weird. If you don't pass `--minimal` the full version will be printed. 

By default, [click expects](https://click.palletsprojects.com/en/7.x/options/#boolean-flags) an on-flag and an off-flag for boolean flags, so they translate to `True/False` and `default=False` -> we pass `--full` by default.

Reply:
Maybe a better option for clarity would be to make it a flag:
```
@click.option('--minimal', is_flag=True, default=False,
              help='Only show minimal curate report.')
```

Reply:
That would be possible but throughout archery the double-flag syntax is used:
![image](https://user-images.githubusercontent.com/16141871/171639959-610c8760-0937-4262-84b2-7abe9dee9c1a.png)
I think it would be better to stick to that convention for consistency.

Reply:
We use both already :)

![image](https://user-images.githubusercontent.com/639755/171641073-0400f035-2b69-4e27-97d2-f5334d4bb5d5.png)

Reply:
It's really just a confusing feature of click, so +1 on `is_flag`

Reply:
Since we have multiple occurrences this can be done in a follow-up.